cmd/apps.go

```diff
@@ -104,11 +106,19 @@ func AppInfo(appID string) error {
 		return err
 	}
 
+	url, err := appURL(s, appID)
+	if checkAPICompatibility(s.Client, err) != nil {
+		if err != errNoDomain {
+			return err
+		}
+		url = errNoDomain.Error()
+	}
+
 	fmt.Printf("=== %s Application\n", app.ID)
 	fmt.Println("updated: ", app.Updated)
 	fmt.Println("uuid:    ", app.UUID)
 	fmt.Println("created: ", app.Created)
-	fmt.Println("url:     ", app.URL)
+	fmt.Println("url:     ", url)
```

Comment:
I feel like we shouldn't be changing this because the server fills out this field for us. If the server is returning us false information when the app has no domain, then we should fix this in the controller.